src/src/AdminBundle/Resources/views/Security/login.html.twig

```diff
@@ -0,0 +1,22 @@
+{% extends "@AdminBundle/Resources/views/base.html.twig" %}
+
+{% block content %}
+    <div style="margin-top: 100px;" class="col-md-4 offset-md-4">
+        {% if error %}
+            <div class="alert alert-danger" role="alert">{{ error.messageKey|trans(error.messageData, 'security') }}</div>
```

Comment:
len tak zo zaujimavosti, co robi `|trans(error.messageData, 'security')` ?

Reply:
Do error objektu mozu prist rozdielne error classy, takto to bolo napisane v symfony navode pre robenie formu.

https://stackoverflow.com/questions/30615754/symfony2-difference-between-error-message-and-error-messagekey